amd_openvx_extensions/amd_rpp/CMakeLists.txt

```diff
@@ -212,7 +216,7 @@ if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
     set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} /MTd")
 else()
     set(LINK_LIBRARY_LIST ${LINK_LIBRARY_LIST} openvx)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fdata-sections -std=gnu++14")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fdata-sections -std=gnu++14 -mavx2")
```

Comment:
why do we need this avx flag for amd_rpp exentensions?

Reply:
This flag is needed for the AVX2 code used in TensorTensorAdd and TensorMulScalar openvx nodes.